tests/unit/init/init_core.js

```diff
@@ -111,7 +111,7 @@ require( [
 				function() {
 					var firstPage = findFirstPage();
 
-					assert.deepEqual( $.mobile.pageContainer[ 0 ], firstPage.parent()[ 0 ] );
+					assert.deepEqual( $( ".ui-pagecontainer" )[ 0 ], firstPage.parent()[ 0 ] );
```

Comment:
I guess this is OK for testing purposes since we control the content and we know there is only one pagecontainer. Still, maybe $.mobile.pagecontainers.active is in order?